src/lib/drivers/tcp/server.rs

```diff
@@ -206,21 +213,37 @@ impl DriverInfo for TcpServerInfo {
         let first_schema = &self.valid_schemes()[0];
         let second_schema = &self.valid_schemes()[1];
         vec![
-            format!("{first_schema}://<IP>:<PORT>").to_string(),
-            url::Url::parse(&format!("{first_schema}://0.0.0.0:14550"))
-                .unwrap()
+            format!("{first_schema}://<IP>:<PORT>?direction=<DIRECTION|receiver,sender>")
                 .to_string(),
-            url::Url::parse(&format!("{second_schema}://127.0.0.1:14660"))
+            url::Url::parse(&format!("{first_schema}://0.0.0.0:14550"))
```

Comment:
```suggestion
            url::Url::parse(&format!("{first_schema}://0.0.0.0:14660"))
```

Reply:
the ip is also changing. Is this intentional?

Reply:
you are looking into the wrong line.
The original code was
```rs
url::Url::parse(&format!("{first_schema}://0.0.0.0:14550"))
```

Reply:
![image](https://github.com/user-attachments/assets/10b2bca4-3851-4b74-b951-8ed2573e396c)

Reply:
I meant: the original port was `14660`, and we are now changing it to `14550`, which makes it inconsistent with the code at lines 221~225, which uses `14660`:

```rust
           url::Url::parse(&format!(
                "{second_schema}://127.0.0.1:14660?direction=receiver"
            ))
            .unwrap()
            .to_string(),
```

it looked like a copy-paste mistake, that's why I'm asking if that's intentional

Reply:
The code is the same, it's only the diff that looks that the numbers changed:

![image](https://github.com/user-attachments/assets/77a17c31-c509-4bc5-812b-a671cbef25ba)